src/pages/Home/About/AboutIntro/About.module.scss

```diff
@@ -1,48 +1,105 @@
 @import url("https://fonts.googleapis.com/css2?family=Teko:wght@300;400;500;600&display=swap");
```

Comment:
I think this can probably go in the index css file rather than here

Reply:
Thanks. I moved it to globals.